subworkflows/local/vcf_phase_panel/main.nf

```diff
@@ -0,0 +1,40 @@
+include { VCF_PHASE_SHAPEIT5                     } from '../../../subworkflows/nf-core/vcf_phase_shapeit5/main'
+
+workflow VCF_PHASE_PANEL {
+    take:
+    ch_vcf          // channel: [ [id, chr], vcf, index ]
+    ch_panel_norm
+    ch_panel_sites
+    ch_panel_tsv
+
+    main:
+
+    ch_versions = Channel.empty()
+
+    // Phase panel
+    if (params.phased == false) {
+        VCF_PHASE_SHAPEIT5(ch_vcf
+            .map { meta, vcf, csi -> [meta, vcf, csi, [], meta.region] },
+        Channel.of([[],[],[]]).collect(),
+        Channel.of([[],[],[]]).collect(),
+        Channel.of([[],[]]).collect())
+        ch_versions = ch_versions.mix(VCF_PHASE_SHAPEIT5.out.versions)
+        ch_panel_phased = VCF_PHASE_SHAPEIT5.out.variants_phased
+            .combine(VCF_PHASE_SHAPEIT5.out.variants_index, by: 0)
+    } else {
+        ch_panel_phased = ch_vcf
+    }
+
+    ch_panel = ch_panel_norm
```

Comment:
I think this combination of channels should be made in the main script.
This subworkflow should only phase the panel (maybe the if statement should also be in the main script).

Reply:
Mmm, the issue I see is that if the subworkflow is only to phase the panel, then it would be the same as `VCF_PHASE_SHAPEIT5`.